sqlmesh/core/macros.py

```diff
@@ -72,6 +72,13 @@ class MacroStrTemplate(Template):
 EXPRESSIONS_NAME_MAP = {}
 SQL = t.NewType("SQL", str)
 
+SUPPORTED_TYPES = {
+    "t": t,
```

Comment:
Does this work if they do 'typing.List'? If not want to support that too?

Reply:
added it